travis/test.sh

```diff
@@ -21,6 +21,8 @@ flutter test --flutter-repo
 
 if [ $TRAVIS_PULL_REQUEST = "false" ]; then
   if [ $TRAVIS_BRANCH = "master" ]; then
+    pub global activate dartdoc 0.8.4
+    export PATH="$PATH":"~/.pub-cache/bin"
```

Comment:
I'm pretty certain this exists earlier. I definitely see it in the environment for travis.

I'm confused how we don't already have the latest version of dartdoc?